### ISSUE TITLE
[Request] 建议设置分辨率的滑动条，可以设置为关闭，手机上很容易误碰到，并且占用很大面积。

### ISSUE THREAD

**Issue by ccdanpian** [lobehub/sd-webui-lobe-theme#502]
### 🥰 需求描述 | Feature Description

其实这写滑动条即使在PC侧也没有太大意义，建议默认为关闭，并且调整一下手机上的布局，一些基础参数都放在生成按钮之上。

### 🧐 解决方案 | Proposed Solution

建议默认为关闭，并且调整一下手机上的布局，一些基础参数都放在生成按钮之上。

**Comment by kaalibro** [comment]
I don't think that the resolution slider is not something necessary, especially that it is worth hiding it by default.
You can customize interface using the AUTO1111 WebUI functionality.

_Option one:_

1) **Settings** → **UI alternatives** → uncheck the **Show Width/Height and Batch sliders in same row**
2) **Settings** → **User interface** → **UI item order for txt2img/img2img tabs** → select items in the order you want and lastly select "dimensions" and "scripts".

_Option two:_

Add some styles like this to the [user.css](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features#usercss) to remove the slider from the UI completely:
```
#txt2img_column_size,
#txt2img_dimensions_row {
  display: none;
}
```

**Comment by ccdanpian** [comment]
@kaalibro 我把option1和option2都试了一下，重启后，width和height的设置还都在。

**Comment by kaalibro** [comment]
> 不过没有找到user.css, 在style.css中增加的

You should not make changes to "style.css" as the `git pull` will overwrite it. "user.css" [should be created](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features#usercss) if it doesn't exist.

![image](https://github.com/lobehub/sd-webui-lobe-theme/assets/44464226/e40ebede-dc1f-4508-b9ca-5663ff91a357)

**Comment by ccdanpian** [comment]
@kaalibro 确实生效了，不过不仅仅slider隐藏了，输入框也不见了，没有办法去设置宽高尺寸了。

**Comment by kaalibro** [comment]
This will hide only the sliders on a mobile layout:

_Option one_
```
/* Remove specified sliders only */
@media (max-width: 575px) {
  [id$='img_column_size'] input[type='range'],
  [id$='img_batch_count'] input[type='range'],
  [id$='img_batch_size'] input[type='range'],
  [id$='img_cfg_scale'] input[type='range'],
  [id$='img_steps'] input[type='range'],
  [id='img2img_denoising_strength'] input[type='range'] {
    display: none;
  }
}
```

_Option two_
```
/* Remove all sliders */
@media (max-width: 575px) {
  input[type='range'] {
    display: none;
  }
}
```

**Comment by ccdanpian** [comment]
@kaalibro Thanks very much，it's worked.

**Comment by ccdanpian** [comment]
建议能够便捷的配置显示或者取消显示这个滑动条，还是很不方便。
